_sass/_cards.scss

```diff
@@ -18,44 +16,6 @@
     }
   }
 
-  .corner-tag {
```

Comment:
@saracope , is there a reason that the corner tag is being deleted?  I believe this rule is necessary to display the data quality score for the cards, which you can view here: https://gsa.github.io/code-gov-style/components/corner_tags

Reply:
No, I see what happened. This code isn't in the local package so when I copied my changes here, it was dropped off.  I will add it back in.